### DISCUSSION TITLE
생애 첫 기술면접 후기!

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#333] [4. 기타 글 모음]
https://shkisme.vercel.app/%EC%84%BC%EB%94%94-%EB%A9%B4%EC%A0%91-%ED%9B%84%EA%B8%B0

처음으로 봤던, 생애 첫 기술면접의 후기를 작성했습니다! 그런데 쓰다 보니 회고 느낌의 글이 되었네용. 면접을 본 회사는 스타트업 센디이고, 직무는 백엔드입니다!

+) 그간 이것 저것 바빠서 글 못썼습니다...
죄!송!합!니!다

**Comment by 1jeongg** [comment]
합격 축하드려요~!! 내용이 너무 공감가네요

항상 응원하겠습니다 화이팅 ♥️♥️

**Reply by shkisme** [reply]
감사합니당!! ㅎㅎ 조만간 만나서 밥 한번 먹쟈! 🥰

**Comment by 02ggang9** [comment]
프리패스 정말로 축하드립니다~ ㄷㄷ! 내용도 정말 알차네용

저도 응원하겠습니다~ ㅎㅇㅌ!! 👍👍

**Reply by shkisme** [reply]
고마워~~ 짱구 이제 나보다 잘해서 어디든 초고속 프리패스 할듯. ㅋㅋ

앞으로도 키퍼 리뉴얼 더 화이팅해보자! 항상 많이 배워갑니다~~
